src/main/features/response/routes/check-and-send.ts

```diff
@@ -103,19 +102,16 @@ export default express.Router()
         switch (responseType) {
           case ResponseType.OWE_NONE:
             if (defendantIsCounterClaiming(user)) {
-              res.redirect(Paths.counterClaimPage.evaluateUri({ externalId: user.claim.externalId }))
+              res.redirect(Paths.defenceRejectAllOfClaimPage.evaluateUri({ externalId: user.claim.externalId }))
```

Comment:
this isn't right its going to a page with radio buttons, it should still go to the guidance page until we can handle it.